packages/fxa-react/components/Footer/index.tsx

```diff
@@ -17,6 +26,8 @@ export const Footer = () => {
       <LinkExternal
         href="https://www.mozilla.org/about/?utm_source=firefox-accounts&utm_medium=Referral"
         data-testid="link-mozilla"
+        // Using grey outline here to match the rest of the footer
```

Comment:
Could we switch the focus outline to our standard blue for consistency across the app? If there’s a specific reason to match the component’s current style instead, let me know and we can discuss.